ui/widget.js

```diff
@@ -246,7 +257,7 @@ $.widget.bridge = function( name, object ) {
 	};
 };
 
-$.Widget = function( /* options, element */ ) {};
+$.Widget = function( /* Options, element */ ) {};
```

Comment:
@scottgonzalez just to be clear JSCS did not capitalize it I did, trying to get this passing. I meant to ( and did before landing ) revert them. JSCS only warns on first letter capitalization. The changes for this in the PR for comment capitalization were done with a search and replace. I was trying to get 100% passage with out having to have this rule exceptions from the jQuery preset. However after finding the examples above i decided this was not possible.